communication-service/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const cors = require("cors");
+const http = require("http");
+
+const app = express();
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(cors()); // config cors so that front-end can use
+app.options("*", cors());
+
+const httpServer = http.createServer(app);
+const io = require("socket.io")(httpServer, {
+  cors: {
```

Comment:
Can you add another attribute called `path: "/communication-api"` above line 14.

Used in deployment for frontend to differentiate the various services (as all the services has the same base url).

_Can reference matching-service in siqi's PR https://github.com/CS3219-AY2223S1/cs3219-project-ay2223s1-g8/pull/64_

Reply:
fixed.